spec/services/stats/r402_offender_sar_complaint_closed_cases_report_spec.rb

```diff
@@ -58,7 +63,10 @@ module Stats
 
       context '#case_scope' do
         it 'ignores any selected periods' do
-          expected = [@closed_offender_sar.name, @closed_offender_sar1.name]
+          expected = [
+            @closed_standard_complaint.name, 
+            @closed_ico_compliant.name, 
+            @closed_litgation_complaint.name]
```

Comment:
I don't think we should hold up this PR, but fwiw as a general approach it makes me nervous seeing instance variables in specs - I think these should almost always be `let` rather than instance variables - maybe something to discuss as a group?